bridge/jsb_module_resolver.cpp

```diff
@@ -372,11 +380,11 @@ namespace jsb
                     {
                         const String dot = ".";
                         const String dot_slash = "./";
-                        const String& main = package_exports[key_main];
+                        const String& main = package_json[key_main];
```

Comment:
Looks like this snuck in from the other PR.

Reply:
good catch removed it